mgxs-part-i.ipynb

```diff
@@ -345,7 +350,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 12,
    "metadata": {},
    "outputs": [],
    "source": [
```

Comment:
It looks like the lines below (for some reason I can't comment on them?) use `groups=` where it should be `energy_groups=` since the constructors (e.g.[ TotalXS constructor](https://docs.openmc.org/en/stable/pythonapi/generated/openmc.mgxs.TotalXS.html)) for `mgxs.TotalXS`, `mgxs.AbsorptionXS`, and `mgxs.ScatterXS` do not have a parameter `groups`. I haven't gone through all the other files, but I wouldn't be surprised if this happened in other places too. Maybe a `grep` would help find all the instances that are `groups` but should be switched to `energy_groups`

Reply:
```
    "total = mgxs.TotalXS(domain=cell, groups=groups)\n",
    "absorption = mgxs.AbsorptionXS(domain=cell, groups=groups)\n",
    "scattering = mgxs.ScatterXS(domain=cell, groups=groups)\n",
```
^ These ones

Reply:
Thanks for the heads up @lewisgross1296! Working on updating those now.